api/auth/center.go

```diff
@@ -209,7 +211,9 @@ func (c *center) Authenticate(r *http.Request) (*Box, error) {
 		return nil, err
 	}
 
-	if hdr := r.Header.Get(ContentEncodingHdr); hdr == ContentEncodingAwsChunked {
+	amzContent := r.Header.Get(AmzContentSha256)
+
+	if hdr := r.Header.Get(ContentEncodingHdr); hdr == ContentEncodingAwsChunked || amzContent == ContentEncodingChunked {
```

Comment:
I'd change `hdr` here to something more meaningful then (it's about content encoding).